packages/compass-connections-navigation/src/virtual-list/use-virtual-navigation-tree.tsx

```diff
@@ -130,7 +130,7 @@ export function useVirtualNavigationTree<T extends HTMLElement = HTMLElement>({
   activeItemId?: string;
   onExpandedChange(item: VirtualTreeItem, isExpanded: boolean): void;
   onFocusMove?: (item: VirtualTreeItem) => void;
-}): [React.HTMLProps<T>, string | undefined] {
+}): [React.HTMLProps<T>, string | undefined, boolean] {
   const rootRef = useRef<T | null>(null);
   const activeId = activeItemId || findFirstItem(items)?.id;
```

Comment:
I'm not sure why, but this findFirstItem bit is now broken

https://github.com/mongodb-js/compass/assets/5196720/c777bad3-1360-45e4-9c93-df9c0416f124

Reply:
correction: actually it's broken on main already and it's specifically after databases are selected, so not related to this PR

Reply:
https://github.com/mongodb-js/compass/pull/5928